go/nbs/mem_table.go

```diff
@@ -79,6 +79,20 @@ func (mt *memTable) getMany(reqs []getRecord, wg *sync.WaitGroup) (remaining boo
 	return
 }
 
+func (mt *memTable) extract(order EnumerationOrder, chunks chan<- extractRecord) {
+	sort.Sort(hasRecordByOrder(mt.order)) // ensure "insertion" order for extraction
```

Comment:
I think it will be in insertion order, but we don't document that as a requirement, so I was programming defensively. I can just require that that be an invariant